iceoryx_posh/test/moduletests/test_mepoo_shared_chunk.cpp

```diff
@@ -59,134 +61,126 @@ class SharedChunk_Test : public Test
     MemPool chunkMgmtPool{64U, 10U, allocator, allocator};
     void* memoryChunk{mempool.getChunk()};
     ChunkManagement* chunkManagement = GetChunkManagement(memoryChunk);
-    iox::mepoo::SharedChunk sut{chunkManagement};
+    SharedChunk sut{chunkManagement};
 };
 
-TEST_F(SharedChunk_Test, DefaultCTor)
+TEST_F(SharedChunk_Test, SharedChunkObjectUpOnInitilizationSetsTheChunkHeaderToNullPointer)
 {
-    EXPECT_THAT(mempool.getUsedChunks(), Eq(1));
-    EXPECT_THAT(chunkMgmtPool.getUsedChunks(), Eq(1));
-}
+    SharedChunk sut;
 
-TEST_F(SharedChunk_Test, CopyCTor)
-{
-    {
-        iox::mepoo::SharedChunk sut2(sut);
-    }
-    EXPECT_THAT(mempool.getUsedChunks(), Eq(1));
-    EXPECT_THAT(chunkMgmtPool.getUsedChunks(), Eq(1));
+    EXPECT_THAT(sut.getChunkHeader(), Eq(nullptr));
 }
 
-TEST_F(SharedChunk_Test, MoveCTor)
+TEST_F(SharedChunk_Test, VerifyCopyConstructorOfSharedChunk)
 {
-    {
-        iox::mepoo::SharedChunk sut2(std::move(sut));
-    }
-    EXPECT_THAT(mempool.getUsedChunks(), Eq(0));
-    EXPECT_THAT(chunkMgmtPool.getUsedChunks(), Eq(0));
+    SharedChunk sut1(sut);
+
+    EXPECT_EQ((sut1.getChunkHeader())->chunkSize(), (sut.getChunkHeader())->chunkSize());
```

Comment:
I think this is not a sufficient check for the copy c'tor. The `chunkSize` could be the same for other reasons. Isn't it possible to check whether the `ChunkManagement*` is the same via comparing `sut.release()` and `sut1.release()`?